### DISCUSSION TITLE
Can you using Flynt with WordPress Multisite and parent/child themes?

### DISCUSSION THREAD

**Discussion by duncanmorley** [flyntwp/flynt#433] [Q&A] [answered]
Hi there

We love the Flynt framework and have used it very successfully on a few single site installations using a parent theme.

Does anyone know if it's possible to use Flynt within a WordPress multisite setup using parent and child themes? If anyone has had any experience using it successfully with WP multisite then It'd be great to know and if there were any particular challenges you encountered.

Many thanks

**Comment by aaronmeder** [comment, marked as answer]
Hey @Skweeze 

you can surely use Flynt on a multisite, but as far as I know not with a parent / child setup.

How I usually approach this:
if the changes from site to site that use the same theme are only marginal I will try to stay with the same theme and enable the needed customization through Flynt's Global/Translatable Options or a customizer setting. You can then for example load or not load some parts of the theme (like a custom post type) based on these settings or customize the design, logo or texts.

If the changes from site to site are going to be larger then I would copy the theme (has it's downsides too of course).

Best
Aaron

**Reply by duncanmorley** [reply]
Thanks so much for the answer. We'll give it a go.
